src/main/java/org/matsim/analysis/postAnalysis/drt/DrtServiceQualityAnalysis.java

```diff
@@ -0,0 +1,108 @@
+package org.matsim.analysis.postAnalysis.drt;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVPrinter;
+import org.apache.commons.csv.CSVRecord;
+import org.matsim.analysis.postAnalysis.traffic.TrafficAnalysis;
+import org.matsim.api.core.v01.Id;
+import org.matsim.api.core.v01.TransportMode;
+import org.matsim.api.core.v01.network.Link;
+import org.matsim.api.core.v01.network.Network;
+import org.matsim.application.MATSimAppCommand;
+import org.matsim.contrib.common.util.DistanceUtils;
+import org.matsim.core.config.Config;
+import org.matsim.core.config.ConfigUtils;
+import org.matsim.core.network.NetworkUtils;
+import org.matsim.core.router.costcalculators.RandomizingTimeDistanceTravelDisutilityFactory;
+import org.matsim.core.router.speedy.SpeedyALTFactory;
+import org.matsim.core.router.util.LeastCostPathCalculator;
+import org.matsim.core.router.util.TravelDisutility;
+import org.matsim.core.router.util.TravelTime;
+import picocli.CommandLine;
+
+import java.io.FileWriter;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@CommandLine.Command(
+        name = "analyze-drt-service",
+        description = "Analyze DRT service quality"
+)
+public class DrtServiceQualityAnalysis implements MATSimAppCommand {
+    @CommandLine.Option(names = "--network", description = "path to network file", required = true)
+    private String networkFile;
+
+    @CommandLine.Option(names = "--events", description = "path to events file", required = true)
+    private String eventsFile;
```

Comment:
For file names, please use Path instead of just String.  This is much more useful when working with functions outside MATSim.

Reply:
Okay.